modules/adnuntiusBidAdapter.js

```diff
@@ -1,38 +1,50 @@
 import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
 
 const BIDDER_CODE = 'adnuntius';
 const ENDPOINT_URL = 'https://delivery.adnuntius.com/i?tzo=';
+const GVLID = 855;
 
 export const spec = {
   code: BIDDER_CODE,
-
+  gvlid: GVLID,
+  supportedMediaTypes: [BANNER],
   isBidRequestValid: function (bid) {
     return !!(bid.bidId || (bid.params.member && bid.params.invCode));
   },
 
-  buildRequests: function (validBidRequests) {
+  buildRequests: function (validBidRequests, bidderRequest) {
     const networks = {};
     const bidRequests = {};
     const requests = [];
+    const segments = config.getConfig('segments');
+    utils.logMessage('CONF', segments)
```

Comment:
I think this line should be removed entirely, as this does not seem like a necessary log to me. I believe that we would rather not inflate code with messaging that is not pertinent as this is neither a user warning/error or really clear in label as to what is being logged - also this can be achieved in the console via a pbjs.getConfig('segments').

Reply:
yeaj, that was just a blunder from me, will remove that.